contrib/wheel_add_ucx_plugins.py

```
@@ -163,21 +163,65 @@ def get_lib_deps(lib_path):
+
     Returns:
-        List of files copied.
+        Tuple of (copied_files, dedup_map):
+        - copied_files: list of absolute destination paths that were copied.
+        - dedup_map: dict mapping removed filenames to the kept filename,
+          e.g. ``{"libuct_ib.so": "libuct_ib.so.0.0.0",
+                  "libuct_ib.so.0": "libuct_ib.so.0.0.0"}``.
     """
     copied_files = []
+    dedup_map = {}
     for root, dirs, files in os.walk(src):
```

Ruff (0.15.10)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)